### PR TITLE
Update aws-lambda-java-log4j dependency to 1.0.1 which contains log4j…

### PR DESCRIPTION
**Pull request by disa6302**
… update

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -96,7 +96,7 @@
         <dependency>
             <groupId>com.amazonaws</groupId>
             <artifactId>aws-lambda-java-log4j</artifactId>
-            <version>1.0.0</version>
+            <version>1.0.1</version>
```
Comment (hassanctech) [inline pom.xml:99]: So this doesn't contain any fix this is from 2020, according to this: https://github.com/aws/aws-lambda-java-libs/tree/master/aws-lambda-java-log4j2 we should be using `1.5.1`

Comment (disa6302) [reply pom.xml]: My bad. Fixed.

**Review by unicornss — APPROVED** [review]

**Review by hassanctech — COMMENTED** [review]
Can you just confirm that a local maven build works with this change?  I think it should but I wonder if there are any other dependencies we need to update as well.

File: pom.xml [hunk 2]
```diff
@@ -96,7 +96,7 @@
         <dependency>
             <groupId>com.amazonaws</groupId>
             <artifactId>aws-lambda-java-log4j</artifactId>
```
Comment (hassanctech) [inline pom.xml]: Discussed offline `aws-lambda-java-log4j` doesn't have a version 1.5.1, that version is in this package: `aws-lambda-java-log4j2`

Comment (disa6302) [reply pom.xml]: Tested the build. It runs clean with `aws-lambda-java-log4j2` version 1.5.1

Comment (hassanctech) [reply pom.xml]: yayyyy

**Review by hassanctech — APPROVED** [review]
lgtm!

**Pull request merged**